tools/src/main/java/com/capitalone/dashboard/tools/App.java

```diff
@@ -0,0 +1,77 @@
+package com.capitalone.dashboard.tools;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.OptionBuilder;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+import com.capitalone.dashboard.tools.utils.EncryptionTool;
+import com.capitalone.dashboard.util.EncryptionException;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+	
+	Options options = new Options();
+    public static void main( String[] args )
+    {
+        App app = new App();
+        CommandLine line = null;
+        try {
+			line = app.parseArgs(args);
+		} catch (ParseException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
probably should fail / exit right? not much can be done in this case. a more meaningful message would be better too..